### PR TITLE
Feature/add find event user endpoint

### PR DESCRIPTION
**Pull request by liza459**
https://trello.com/c/wSF2wLnJ/204-eventservice-add-findeventuser-endpoint

Also:
- corrected namespace in Startup and Program;
- fixed bug in CreateEventUserCommand (now returned "true" on successful response);
- remove unused using in CreateEventCategoryCommand and CreateEventCategoryRequestValidator.

### CODE REVIEW

File: src/EventService.Models.Dto/Requests/EventUser/FindEventUsersFilter.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+public record FindEventUsersFilter : BaseFindFilter
+{
+  [FromQuery(Name = "userFullNameIncludeSubstring")]
+  public string UserFullNameIncludeSubstring { get; set; }
+
+  [FromQuery(Name = "status")]
+  public EventUserStatus? Status { get; set; }
+
+  [FromQuery(Name = "isAscendingSort")]
+  public bool? IsAscendingSort { get; set; } = null;
```
Comment (vorobevamv) [inline src/EventService.Models.Dto/Requests/EventUser/FindEventUsersFilter.cs]: don't need '= null' - bool? is null by default

File: src/EventService.Data/EventUserRepository.cs [hunk 2]
```diff
@@ -34,4 +37,20 @@ public async Task<bool> CreateAsync(List<DbEventUser> dbEventUsers)
...
     return true;
   }
+
+  public Task<List<DbEventUser>> FindAsync(
+    Guid eventId,
+    FindEventUsersFilter filter,
+    CancellationToken cancellationToken)
+  {
+    IQueryable<DbEventUser> eventUsersQuery = _provider.EventsUsers.Where(eu =>
+      eu.EventId == eventId).AsQueryable();
```
Comment (vorobevamv) [inline src/EventService.Data/EventUserRepository.cs]: don't need 'AsQueryable()' - it's already IQueryable
add AsNoTracking() after _provider.EventsUsers

File: src/EventService.Business/Commands/EventUser/FindEventUserCommand.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    if (eventUsers is null || !eventUsers.Any())
+    {
+      return new();
+    }
+
+    IFilteredUsersDataResponse filteredUsersData = await _userService.FilteredUsersDataAsync(
+      usersIds:eventUsers.Select(e => e.UserId).ToList(), 
+      skipCount: filter.SkipCount, 
+      takeCount: filter.TakeCount, 
+      ascendingSort: filter.IsAscendingSort,
```
Comment (vorobevamv) [inline src/EventService.Business/Commands/EventUser/FindEventUserCommand.cs]: remove spaces after ','

File: src/EventService.Business/Commands/EventUser/FindEventUserCommand.cs [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    CancellationToken cancellationToken)
+  {
+    if (!_baseFindFilterValidator.ValidateCustom(filter, out List<string> errors) || 
+      !await _eventRepository.DoesExistAsync(eventId))
+    {
+      return _responseCreator.CreateFailureFindResponse<EventUserInfo>(HttpStatusCode.BadRequest);
+    }
+
+    List<DbEventUser> eventUsers =
+        await _eventUserRepository.FindAsync(eventId: eventId, filter: filter, cancellationToken: cancellationToken);
```
Comment (VLiamin) [inline src/EventService.Business/Commands/EventUser/FindEventUserCommand.cs]: 4 spaces insted of 2

File: src/EventService.Data/EventUserRepository.cs [hunk 5]
```diff
@@ -34,4 +37,20 @@ public async Task<bool> CreateAsync(List<DbEventUser> dbEventUsers)
...
+
+    if (filter.Status != null)
```
Comment (VLiamin) [inline src/EventService.Data/EventUserRepository.cs]: is not null

**Review by Segway-svg — APPROVED** [review]

**Comment by TheDioBrando** [comment]
replace the brackets with a semicolon at namespaces in some files

File: src/EventService.Business/Commands/EventUser/FindEventUserCommand.cs [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+    List<DbEventUser> eventUsers =
+      await _eventUserRepository.FindAsync(eventId: eventId, filter: filter, cancellationToken: cancellationToken);
+
+    if (eventUsers is null || !eventUsers.Any())
+    {
+      return new();
+    }
+
+    (List<UserData> usersData, int TotalCount) = await _userService.FilteredUsersDataAsync(
```
Comment (vorobevamv) [inline src/EventService.Business/Commands/EventUser/FindEventUserCommand.cs]: int totalCount :)

**Review by vorobevamv — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

**Review by Draktolli — APPROVED** [review]

**Review by TheDioBrando — APPROVED** [review]

**Pull request merged**